STMReco/fcl/plotSTMWaveformDigis.fcl

```diff
@@ -4,18 +4,20 @@
 
 #include "Offline/fcl/standardServices.fcl"
 
-process_name: PlotSTMSpectrum
+process_name: PlotSTMWaveformDigis
 
 source : {
   module_type : RootInput
 }
 
-services : @local::Services.Reco
+services : {
+  message : @local::Services.Core.message
+  TFileService : { }
```

Comment:
If you don't need TFileService, don't include it.

Reply:
For plotSTMWaveformDigis.fcl, I do use but I define the file name later. In commit e102cd663 I moved the file name into this block to make it obvious